io.elementary.code.json

```diff
@@ -0,0 +1,247 @@
+{
+	"app-id": "io.elementary.code",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.24",
+	"sdk": "org.gnome.Sdk",
+	"command": "scratch-text-editor",
+	"rename-desktop-file": "org.pantheon.scratch.desktop",
+	"rename-appdata-file": "org.pantheon.scratch.appdata.xml",
+	"finish-args": [
+		"--filesystem=host",
+		/* Wayland and X11 access */
```

Comment:
`flatpak-builders` implementation is `json-glib` which does allow them. I think we generally do want to avoid them though so other tools can more reasonably manipulate them.